tools/mcp/azure-sdk-python-mcp/main.py

```diff
@@ -0,0 +1,247 @@
+from mcp.server.fastmcp import FastMCP
```

Comment:
We have been trying to keep all the engsys related tools under the eng folder instead of the root tools folder. Scott has been trying to eliminate it but hasn't had cycles to finish the last one.